lib/endpoints/vms.js

```diff
@@ -213,6 +218,18 @@ function listVms(req, res, next) {
 
             res.header('x-joyent-resource-count', count);
 
+            var limit = DEFAULT_LIST_VM_LIMIT;
+            if (req.params.limit)
+                limit = Number(req.params.limit);
+
+            params.limit = Math.min(limit, MAX_LIST_VMS_LIMIT);
```

Comment:
Addressed [by the latest commit in this PR's branch](https://github.com/misterdjules/sdc-vmapi/commit/5d65624f5bf76038e74fd8c831bcf56a6f42102f#diff-119842d7ab8075dd26e505403da01facR1370)